templates/gift_card.liquid

```diff
@@ -19,7 +19,8 @@
       <link rel="preconnect" href="https://fonts.shopifycdn.com" crossorigin>
     {%- endunless -%}
 
-    {%- assign formatted_initial_value = gift_card.initial_value | money_without_trailing_zeros: gift_card.currency | strip_html -%}
```

Comment:
Passing in currency as an arg to the money filter is deprecated. Cleaning that up here.